events/babylon_discord_break_1_1_event.txt

```diff
@@ -0,0 +1,413 @@
+﻿namespace = ideology_break_1_1_event
+# Ideological break for level 1 reached
+ideology_break_1_1_event.1 = {
+	type = country_event
+	title = ideology_break_1_1_event.1.t
+	desc = ideology_break_1_1_event.1.d
+	flavor = ideology_break_1_1_event.1.f
+	event_image = {
+		video = "gfx/event_pictures/asia_politician_parliament_motion.bk2"
+	}
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/unspecific/leader_speaking_to_a_group_of_people"
+	duration = 3
+	trigger = {
+		# Caused when a nation hits 100 on their cohesion level.
```

Comment:
chance to be caused, otherwise you would just push a trigger here.

Reply:
fixing it